src/components/EntityPage/File/fileProperties.js

```diff
@@ -10,6 +11,18 @@ export const toFilePropertiesSummary = data => {
   const study = participants.study;
   const biospecimens = participants.biospecimens.hits.edges[0].node;
 
+  // it could have been easier (see below), but `data.experiment_strategies` is sometimes empty,
+  //  even if data.sequencing_experiments.hits.edges is ont empty for the given file 😕
+  // const experimentalStrategies =
+  //   _.uniq(_.get(data, 'experiment_strategies').filter(datum => !!datum)).join(', ') || '--';
```

Comment:
I think we can lose this comment, the `data.experiment_strategies` field is to be removed anyway :)